robottelo/ui/computeresource.py

```diff
@@ -143,9 +143,125 @@ def update(self, name, newname=None, parameter_list=None,
 
     def delete(self, name, really=True):
         """Removes the compute resource entity"""
+        Navigator(self.browser).go_to_compute_resources()
         self.delete_entity(
             name,
             really,
             locators['resource.delete'],
             drop_locator=locators['resource.dropdown'],
         )
+
+    def go_to_compute_resource(self, res_name):
+        """ Navigates to compute resource page """
+        resource = self.search(res_name)
+        if resource is None:
+            raise UINoSuchElementError(
+                'Could not find the resource {0}'.format(res_name))
+        strategy, value = locators['resource.get_by_name']
+        locator = (strategy, value % res_name)
+        self.click(locator, wait_for_ajax=False)
```

Comment:
Do you have specific reason for `wait_for_ajax=False`. Usually, we put that flag to `False` before Javascript alerts as otherwise entire run will fail (We should not have any commands before alert handler)